service/scaladsl/kafka/client/src/main/scala/com/lightbend/lagom/internal/scaladsl/broker/kafka/ScaladslKafkaSubscriber.scala

```diff
@@ -106,52 +99,29 @@ private[lagom] class ScaladslKafkaSubscriber[Payload, SubscriberPayload](
       new ScaladslKafkaDeserializer(deserializer)
     }
 
-    ConsumerSettings(system, keyDeserializer, valueDeserializer)
-      .withBootstrapServers(kafkaConfig.brokers)
+    val config = system.settings.config.getConfig(ConsumerSettings.configPath)
+    ConsumerSettings(config, keyDeserializer, valueDeserializer)
```

Comment:
👍🏼 to using the least powerful constructor of `ConsumerSettings` (I noticed the same improvement with `ProducerSettings`. Nice!